csrc/ir/cloner.h

```diff
@@ -146,16 +146,8 @@ T* IrBuilder::clone(const T* src, IrCloner* ir_cloner) {
   Statement* dest_stmt = dynamic_cast<Statement*>(dest);
 
   auto dest_container = ir_cloner->container();
-  auto src_container = src_stmt->container();
-
   dest_container->registerStmt(IrBuilderPasskey(dest_container), dest_stmt);
-
-  if (src_container != dest_container) {
-    dest_stmt->setName(IrBuilderPasskey(dest_container), src_stmt->name());
```

Comment:
I'm afraid this would make it hard to debug segmentation. Names have been used to correspond segmented fusions with the complete fusion. This change would remove that correspondence. cc @naoyam

Reply:
I see. Let me know then whether to keep this patch (which is in a separate commit, so it's easy to revert)

Reply:
btw I'm seeing from the CI that this commit breaks the unit test `NVFuserTest.FusionCopy_CUDA` which relies on Val's name correspondence.

It may be safer to revert this commit, to de-prioritize printing over core functionalities.